test/GitHubExtension/DeveloperId/LoginUITests.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using GitHubExtension.DeveloperId;
+using GitHubExtension.Test.Mocks;
+using Microsoft.Windows.DevHome.SDK;
+
+namespace GitHubExtension.Test;
+
+// Unit Tests for LoginUIController and LoginUI
+public partial class DeveloperIdTests
+{
+    public const string EMPTYJSON = "{}";
+
+    public struct LoginUITestData
+    {
+        public const string GithubButtonAction = "{\"id\":\"Personal\",\"style\":\"positive\",\"title\":\"Sign in to github.com\",\"tooltip\":\"Opens the browser to log you into GitHub\",\"type\":\"Action.Submit\"}";

```

Comment:
Any reason why you aren't using `$` on the strings.

Reply:
Will fix.